SPECS/keda/keda.spec

```diff
@@ -66,6 +65,11 @@ cp ./bin/keda-admission-webhooks %{buildroot}%{_bindir}
 %{_bindir}/%{name}-admission-webhooks
 
 %changelog
+* Fri Apr 11 2025 Sreeniavsulu Malavathula <v-smalavathu@microsoft.com> - 2.14.1-6
+- Patch CVE-2025-22870 and CVE-2024-51744.
+- Remove redundant untarring from %build section
+- Remove the patch for CVE-2025-29923 as it was causing build failures.
```

Comment:
@v-smalavathu removing a patch because it causes a build failure is not sufficient reason for removing the patch.  If the patch is applying to a file that was patched for secondary reason, then that may be the cause of the build failure and the patch may need to be adjusted not removed.